LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 beta.gouv.fr
```

Comment:
For a future iteration: occurrences of "beta.gouv.fr" in `LICENSE` notices should rather be `beta.gouv.fr, DINSIC`, since `DINSIC` is the legal entity.

Not a blocker for this PR, just a remind-me-later note.

Reply:
I make the change then I merge !